Tasks/AzureStaticWebAppV0/index.ts

```diff
@@ -54,4 +34,66 @@ async function run() {
     }
 }
 
+function createDockerEnvVarFile() {
+    const taskVariables = tl.getVariables();
+    var variableString: string = ""
+
+    const systemVariableNames: Set<string> = new Set<string>();
+
+    const addVariableToString = (variableName: string, variableValue: string) => variableString += variableName + "=" + variableValue + "\n"
+
+    const addSystemVariableToString = (variableName: string, variableValue: string) => {
+        addVariableToString(variableName, variableValue)
+        systemVariableNames.add(variableName)
+    }
+
+    const workingDirectory: string = tl.getInput('cwd', false) || "";
```

Comment:
Not sure if we need to support relative directories here as well. I think most folks are used to passing absolute paths so I think we're okay. Perhaps add a bit more to the help text to say that this is an absolute path.